packages/manager/modules/pci/src/projects/project/instances/instance/instance.constants.js

```diff
@@ -0,0 +1,21 @@
+export const DEDICATED_IPS_URL = {
```

Comment:
Are those supposed to be links to legacy parts of the manager or is this supposed to be sustainable ?

Reply:
Step 1 : add links to legacy part to be able to handle this 
Step 2 : on a second version, determine if we create a module or so MANAGER-2968